core-processor/src/common.rs

```diff
@@ -167,12 +141,12 @@ pub enum JournalNote {
     ///
     /// This should be the last update involving this message id.
     MessageConsumed(MessageId),
-    /// Message was generated.
-    SendMessage {
+    /// Dispatch was generated.
```

Comment:
I would revert this formulation, because actors literally send "messages". It's conversion into dispatch appears under the hood only

Reply:
agreed